### PR TITLE
Network client and movies bloc

### PR DESCRIPTION
**Pull request by esteban027**
Network client working.

### CODE REVIEW

File: lib/src/blocs/movies_bloc.dart [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+
+
+ class LoadMoviesBloc extends Bloc<MoviesEvent, MoviesState> {
+   Repository repository;
+
+
+  @override
+
```
Comment (danielesbe9526) [inline lib/src/blocs/movies_bloc.dart:15]: remove all those blank spaces

File: lib/src/blocs/movies_bloc.dart [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  MoviesState get initialState => MoviesLoading();
+
+  @override
+  Stream<MoviesState> mapEventToState(MoviesEvent event) async* {
+    
+    if (event == MoviesEvent.loadAllMovies) {
+      repository =  Repository();
+      yield* _mapLoadAllMovies();
+          }        
+        }
```
Comment (danielesbe9526) [inline lib/src/blocs/movies_bloc.dart:25]: indentation error

File: lib/src/ui/home_page.dart [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '../blocs/movies_bloc.dart';
+import '../Events/movies_events.dart';
+import '../resources/repository.dart';
+
+class HomePage extends StatelessWidget {
+  const HomePage();
+
+
```
Comment (danielesbe9526) [inline lib/src/ui/home_page.dart:13]: remove all those black spaces

File: lib/src/ui/home_page.dart [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            return Center(child: CupertinoButton(
+              child:Text('Load Movies') ,
+              onPressed: () => moviesBloc.add(MoviesEvent.loadAllMovies),
+              ),
+              );
+          }
+          );
+                  
+
+              break;
```
Comment (danielesbe9526) [inline lib/src/ui/home_page.dart:42]: indentation and spaces

File: lib/src/ui/home_page.dart [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+              );
+              break;
+          }
+          return null;
+        },
+      ),
+    );
+  }
+
+
```
Comment (danielesbe9526) [inline lib/src/ui/home_page.dart:56]: spaces

File: lib/src/ui/home_page.dart [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+          return null;
+        },
+      ),
+    );
+  }
+
+  
+}
+
+// class HomeTabView extends StatelessWidget {
```
Comment (danielesbe9526) [inline lib/src/ui/home_page.dart:59]: remove comment code

File: lib/src/blocs/movies_bloc.dart (hunk 1 above)
Comment (esteban027) [reply lib/src/blocs/movies_bloc.dart:15]: Done

File: lib/src/blocs/movies_bloc.dart (hunk 2 above)
Comment (esteban027) [reply lib/src/blocs/movies_bloc.dart:25]: Done

File: lib/src/ui/home_page.dart (hunk 3 above)
Comment (esteban027) [reply lib/src/ui/home_page.dart:13]: Done

File: lib/src/ui/home_page.dart (hunk 4 above)
Comment (esteban027) [reply lib/src/ui/home_page.dart:42]: Done

File: lib/src/ui/home_page.dart (hunk 5 above)
Comment (esteban027) [reply lib/src/ui/home_page.dart:56]: Done

File: lib/src/ui/home_page.dart (hunk 6 above)
Comment (esteban027) [reply lib/src/ui/home_page.dart:59]: Done

**Review by danielesbe9526 — APPROVED** [review]

**Pull request merged**